packages/lib/gpcircuits/test/proto-pod-gpc.spec.ts

```diff
@@ -20,393 +25,391 @@ import {
-            : BABY_JUB_NEGATIVE_ONE,
-        ownerSemaphoreV3IdentityNullifier: ownerIdentity.nullifier,
-        ownerSemaphoreV3IdentityTrapdoor: ownerIdentity.trapdoor,
-        ownerExternalNullifier: 42n,
-        ownerIsNullfierHashRevealed: isNullifierHashRevealed ? 1n : 0n,
-        globalWatermark: 1337n
-      },
-      outputs: {
-        entryRevealedValueHash: sigEntryRevealedValueHash,
-        ownerRevealedNulifierHash:
-          isNullifierHashRevealed && paramMaxEntries > sigOwnerEntryIndex
-            ? 1517081033071132720435657432021139876572843496027662548196342287861804968602n
-            : BABY_JUB_NEGATIVE_ONE
-      }
-    };
+    // Fill in entry equality rules, which is also how ownership is proven
+    // for multiple objects.  An unconstrained entry is set to be equal
+    // to itself.
+    if (
+      entryInfo.eqEntryIndex !== undefined &&
+      entryInfo.eqEntryIndex < testEntries.length &&
+      entryInfo.eqEntryIndex < paramMaxEntries
+    ) {
+      sigEntryEqualToOtherEntryByIndex.push(BigInt(entryInfo.eqEntryIndex));
+    } else {
+      sigEntryEqualToOtherEntryByIndex.push(BigInt(entryIndex));
+    }
+    sigEntryProofDepth.push(BigInt(entrySignals.proof.siblings.length));
+    sigEntryProofIndex.push(BigInt(entrySignals.proof.index));
+
+    // Fillin sibling array, padded with 0s to max length.
+    sigEntryProofSiblings.push(
+      extendedSignalArray(entrySignals.proof.siblings, paramMaxDepth)
+    );
   }
 
+  return {
+    inputs: {
+      objectContentID: sigObjectContentID,
+      objectSignerPubkeyAx: sigObjectSignerPubkeyAx,
+      objectSignerPubkeyAy: sigObjectSignerPubkeyAy,
+      objectSignatureR8x: sigObjectSignatureR8x,
+      objectSignatureR8y: sigObjectSignatureR8y,
+      objectSignatureS: sigObjectSignatureS,
+      entryObjectIndex: sigEntryObjectIndex,
+      entryNameHash: sigEntryNameHash,
+      entryValue: sigEntryValue,
+      entryIsValueEnabled: testArray2Bits(sigEntryIsValueEnabled),
+      entryIsValueHashRevealed: testArray2Bits(sigEntryIsValueHashRevealed),
+      entryEqualToOtherEntryByIndex: sigEntryEqualToOtherEntryByIndex,
+      entryProofDepth: sigEntryProofDepth,
+      entryProofIndex: sigEntryProofIndex,
+      entryProofSiblings: sigEntryProofSiblings,
+      ownerEntryIndex:
+        paramMaxEntries > sigOwnerEntryIndex
+          ? sigOwnerEntryIndex
+          : BABY_JUB_NEGATIVE_ONE,
+      ownerSemaphoreV3IdentityNullifier: ownerIdentity.nullifier,
+      ownerSemaphoreV3IdentityTrapdoor: ownerIdentity.trapdoor,
+      ownerExternalNullifier: 42n,
+      ownerIsNullfierHashRevealed: isNullifierHashRevealed ? 1n : 0n,
+      globalWatermark: 1337n
+    },
+    outputs: {
+      entryRevealedValueHash: sigEntryRevealedValueHash,
+      ownerRevealedNulifierHash:
+        isNullifierHashRevealed && paramMaxEntries > sigOwnerEntryIndex
+          ? 1517081033071132720435657432021139876572843496027662548196342287861804968602n
```

Comment:
what is the meaning of this big number?

Reply:
It's just the hash which results from the inputs above, because I was too lazy to write the code to calculate it outside the circuit.